dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java

```diff
@@ -0,0 +1,182 @@
+                + "AS\n"
+                + "BEGIN\n"
+                + "WITH cte AS (\n"
+                + "  SELECT TOP(@records_to_fetch) *\n"
+                + "  FROM dist_reindex_journal WITH (ROWLOCK, READPAST, UPDLOCK)\n"
+                + "  WHERE serverid IS NULL\n"
+                + "  AND priority <= @priority_level\n"
+                + "  ORDER BY priority ASC)\n"
+                + "UPDATE cte\n"
+                + "  SET serverid=@server_id\n"
+                + "OUTPUT\n"
+                + "  INSERTED.*\n"
+                + "END;";
+    }
+
+    private String getMySQLScript() {
+        return "CREATE PROCEDURE load_records_to_index(IN server_id VARCHAR(100), IN records_to_fetch INT, IN priority_level INT)\n"
+                + "BEGIN\n"
+                + "DECLARE v_id BIGINT;\n"
+                + "DECLARE v_inode_to_index VARCHAR(100);\n"
+                + "DECLARE v_ident_to_index VARCHAR(100);\n"
+                + "DECLARE v_serverid VARCHAR(64);\n"
+                + "DECLARE v_priority INT;\n"
+                + "DECLARE v_time_entered TIMESTAMP;\n"
+                + "DECLARE v_index_val VARCHAR(325);\n"
+                + "DECLARE v_dist_action INT;\n"
+                + "DECLARE cursor_end BOOL DEFAULT FALSE;\n"
+                + "DECLARE cur1 CURSOR FOR SELECT * FROM dist_reindex_journal WHERE serverid IS NULL or serverid='' AND priority <= priority_level ORDER BY priority ASC LIMIT records_to_fetch;\n"
+                + "DECLARE CONTINUE HANDLER FOR NOT FOUND SET cursor_end:=TRUE;\n"
+                + "\n"
+                + "DROP TEMPORARY TABLE IF EXISTS tmp_records_reindex;\n"
+                + "CREATE TEMPORARY TABLE tmp_records_reindex (\n"
+                + "  id BIGINT PRIMARY KEY,\n"
+                + "  inode_to_index varchar(36),\n"
+                + "  ident_to_index varchar(36),\n"
+                + "  dist_action INT,\n"
+                + "  priority INT\n"
+                + ") ENGINE=MEMORY;\n"
+                + "\n"
+                + "OPEN cur1;\n"
+                + "WHILE (NOT cursor_end) DO\n"
+                + "  FETCH cur1 INTO v_id,v_inode_to_index,v_ident_to_index,v_serverid,v_priority,v_time_entered,v_index_val,v_dist_action;\n"
+                + "  IF (NOT cursor_end) THEN\n"
+                + "    UPDATE dist_reindex_journal SET serverid=server_id WHERE id=v_id;\n"
+                + "    INSERT INTO tmp_records_reindex VALUES (v_id, v_inode_to_index, v_ident_to_index, v_dist_action, v_priority);\n"
+                + "  END IF;\n"
+                + "END WHILE;\n"
+                + "CLOSE cur1;\n"
+                + "\n"
+                + "SELECT * FROM tmp_records_reindex;\n"
+                + "END;";
+    }
+
+    private boolean procedureExists() throws DotDataException {
+        String query = null;
+
+        switch (DbConnectionFactory.getDBType()){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)